src/components/notifications/TechulusPush.vue

```diff
@@ -1,9 +1,40 @@
 <template>
     <div class="mb-3">
-        <label for="push-api-key" class="form-label">{{ $t("API Key") }}</label>
+        <label for="push-api-key" class="form-label">{{ $t("API Key (required)") }}</label>
         <HiddenInput id="push-api-key" v-model="$parent.notification.pushAPIKey" :required="true" autocomplete="new-password"></HiddenInput>
     </div>
 
+    <div class="mb-3">
+        <label for="push-api-title" class="form-label">{{ $t("Title") }}</label>
+        <input id="push-api-title" v-model="$parent.notification.pushTitle" type="text" class="form-control">
+    </div>
+
+    <div class="mb-3">
+        <label for="push-api-channel" class="form-label">{{ $t("Channel") }}</label>
+        <input id="push-api-channel" v-model="$parent.notification.pushChannel" type="text" class="form-control">
+    </div>
+
+    <div class="mb-3">
+        <label for="push-api-sound" class="form-label">{{ $t("Sound") }}</label>
```

Comment:
Please make sure that you have added every translation string to [`en.json`](https://github.com/louislam/uptime-kuma/blob/master/src/lang/en.json). If they are not, they won't be avalibale for translation from our awesome group of translators.

Reply:
Done!